build/Microsoft.DotNet.Cli.Compile.targets

```diff
@@ -124,10 +150,11 @@
                                    AssetPath="$(BinaryToCorehostifyRelDir)/%(BinariesToRemove.Identity).exe" />
 
       <!-- cleanup compilers project output we don't need -->
-      <CleanPublishOutput Path="$(SdkOutputDirectory)"
-                          Name="compilers"
-                          DeleteRuntimeConfigJson="true"
-                          DeleteDepsJson="true" />
+      <Delete Files="$(SdkOutputDirectory)/compilers$(ExeExtension)" />
```

Comment:
Is there anything in the format of compiler.* that you are not deleting?

You could use an ItemGroup and then do Delete on it.